6.crud-php/Models/Model_clients.php

```diff
@@ -0,0 +1,103 @@
+<?php
+require_once("Model.php");
+
+class Model_clients extends Model
+{
+
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    public function getAllClients()
+    {
+        $clients = $this->getDB()
+            ->query("SELECT * FROM  clients")
```

Comment:
don't use select *, specify the columns one by one.
for example

select *
columns_1,
columns_2,
columns_3,

from tbl;